mod-login-saml/src/main/resources/domain/mod-login-saml/features/configureSaml.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Configure SAML
+
+  Background:
+    * url baseUrl
+    * call login testAdmin
+    * configure headers =
+    """
+    {
+      "X-Okapi-Tenant": "#(testTenant)",
+      "X-Okapi-Token": "#(okapitoken)",
+      "Accept": "application/json, text/plain"
+    }
+    """
+  Scenario: Validate IdP url
+    Given path "saml/validate"
+    And param type = "idpurl"
+    And param value = idpUrl
+    When method GET
+    Then status 200
+
+  Scenario: Configure an IdP
+    # Configure the IdP using the same credentials we"ve been using so far. This admin user should have the permissions
+    # based on what we have done in previous steps.
+    Given path "saml/configuration"
+    # Disable the mod-authtoken"s cache. Without this we would need to sleep for > 60 seconds for mod-authtoken
+    # to be aware of the new permissions.
+    And header Authtoken-Refresh-Cache = "true"
+    # Here we need the idpUrl to actually be reachable and return XML.
+    And header Content-Type = "application/json"
+    And request
+    """
+    {
+      "idpUrl": "#(idpUrl)",
+      "samlBinding": "#(binding)",
+      "samlAttribute": "#(samlAttribute)",
+      "userProperty": "#(userProp)",
+      "okapiUrl": "http://localhost:9130"
```

Comment:
Should use `config.baseUrl` configured in karate-config.js

Reply:
Agreed. Using the baseUrl would be better. There are a couple of other places where I use localhost for the okapi URL. I'll change those too.